tgui/packages/tgui/components/TextArea.js

```diff
@@ -126,6 +126,7 @@ export class TextArea extends Component {
     const input = this.textareaRef.current;
     if (input) {
       input.value = toInputValue(nextValue);
+      this.props.textAreaRef(this.textareaRef);
```

Comment:
Should be called innerRef for consistency (inner because it's a ref to the core inner element).

Reply:
👍